commander/src/bootstrapping/commands/generator/export.ts

```diff
@@ -69,6 +69,7 @@ export abstract class ExportCommand extends BaseIPCClientCommand {
 	static description = 'Export to <FILE>.';
 
 	static examples = [
+		'generator:export',
 		'generator:export --output /mypath/genInfo.json',
 		'generator:export --output /mypath/genInfo.json --data-path ./data ',
```

Comment:
It would be nice to update the file name in examples, to also be `generator_info.json`